owners/src/github.js

```diff
@@ -436,7 +435,45 @@ class GitHub {
     );
     this.logger.debug('[listFiles]', number, response.data);
 
-    return response.data.map(({filename, sha}) => filename);
+    return response.data.map(({filename}) => filename);
+  }
+
+  /**
+   * Searches for files in a repo with a given name.
+   *
+   * See https://developer.github.com/v3/search/#search-code
+   *
+   * @param {!string} filename filename to search for.
+   * @return {FileRef[]} list of returned results.
```

Comment:
You seem to be shifting a lot between `string` and `FileRef`. When do you actually need the SHA field?

Reply:
I only need the SHA field when I'm requesting a file's contents from the GitHub API. I'd added FileRef when I thought I'd need to fetch files in a PR, and so I started returning the SHA in listFiles. Then, it was determined we didn't need that, so it became clunky to work with {filename, sha} instead of just strings. Now, I actually want file refs so I can fetch the contents. So, other methods use string or Array<string>, and FileRef is just here for search/fetching file contents